src/web/core/src/main/java/org/geoserver/web/data/layergroup/AbstractLayerGroupPage.java

```diff
@@ -113,6 +116,9 @@ protected void onUpdate(AjaxRequestTarget target) {
         
         form.add(modeChoice);
         
+        queryableCheckBox = new CheckBox("queryable", new Model<Boolean>(!layerGroup.isNotQueryableEnabled()));
```

Comment:
You should be able to directly bind the property once the layer group is property is called "queryable"